src/Simulation/QSharpFoundation/Convert/Convert.qs

```diff
@@ -154,7 +154,7 @@ namespace Microsoft.Quantum.Convert {
     /// Each Pauli operator can be encoded using two bits:
     /// $$
     /// \begin{align}
-    ///     \boldone \mapsto 00, \quad X \mapsto 01, \quad Y \mapsto 11,
+    ///     \quad I \mapsto 00, \quad X \mapsto 01, \quad Y \mapsto 11,
```

Comment:
This change hopefully replaces `1` with `I` on [this page](https://docs.microsoft.com/en-us/qsharp/api/qsharp/microsoft.quantum.convert.pauliarrayasint) (search for "00" or "**1**↦00")

Reply:
$\boldone$ refers to the identity matrix; we've generally used **1** or 𝟙 in preference to 𝐼 for the identity matrix in our docs, such that I'd suggest reverting this change.

Reply:
Reverted